.eslintignore

```diff
@@ -1 +1,3 @@
 coverage
+node_modules
+support
```

Comment:
Interesting enough I get that weird `nodeunit` folder generated as well; TBH, I do not know how it gets generated and that fails lots of eslint rules. @superkhau do you have any idea? I keep deleting that folder.